jobs/rocoto/wavepostbndpnt.sh

```diff
@@ -5,9 +5,65 @@ source "${HOMEgfs}/ush/preamble.sh"
 ###############################################################
 echo
 echo "=============== START TO SOURCE FV3GFS WORKFLOW MODULES ==============="
-. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
-status=$?
-[[ ${status} -ne 0 ]] && exit ${status}
+#. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
+#status=$?
+#[[ ${status} -ne 0 ]] && exit ${status}
+
+# TODO: clean this up
+source "${HOMEgfs}/ush/detect_machine.sh"
+set +x
+source "${HOMEgfs}/ush/module-setup.sh"
+
+
+if [[ "${MACHINE_ID}" != "noaacloud" ]]; then
+  module use "${HOMEgfs}/sorc/ufs_model.fd/tests"
+  module load modules.ufs_model.lua
+  if [[ "${MACHINE_ID}" = "wcoss2" ]]; then
+    module load prod_util
+    module load cray-pals
+    module load libjpeg
+  else
+    module load prod-util
+  fi
+fi
+
+
+if [[ "${MACHINE_ID}" = "hera" ]]; then
+  module use "/scratch2/NCEPDEV/ensemble/save/Walter.Kolczynski/modulefiles/core"
+  module load "miniconda3/4.6.14"
+  module load "gfs_workflow/1.0.0"
+# TODO: orion and wcoss2 will be uncommented when they are ready.  This comment block will be removed in the next PR
+#elif [[ "${MACHINE_ID}" = "orion" ]]; then
+#  module use "/home/rmahajan/opt/global-workflow/modulefiles/core"
+#  module load "python/3.7.5"
+#  module load "gfs_workflow/1.0.0"
+#elif [[ "${MACHINE_ID}" = "wcoss2" ]]; then
+#  module load "python/3.7.5"
+fi
+if [[ "${MACHINE_ID}" == "noaacloud" ]]; then
+   if [[ "${PW_CSP:-}" = "aws" ]]; then
+
+      # TODO: This can be cleaned-up; most of this is a hack for now.
+      module use "/contrib/spack-stack/envs/ufswm/install/modulefiles/Core"
+      module load "stack-intel"
+      module load "stack-intel-oneapi-mpi"
+      module use -a "/contrib/spack-stack/miniconda/modulefiles/miniconda/"
+      module load "py39_4.12.0"
+      module load "ufs-weather-model-env/1.0.0"
+      export NETCDF="/contrib/spack-stack/miniconda/apps/miniconda/py39_4.12.0"
+      # TODO: Are there plans for EPIC to maintain this package or should GW provide support?
+      export UTILROOT="/contrib/global-workflow/NCEPLIBS-prod_util"
+      export PATH="${PATH}:/contrib/global-workflow/bin"
+      ndate_path="$(command -v ndate)"
+      export NDATE="${ndate_path}"
+   fi
+fi
+
+module list
+unset MACHINE_ID
+set_trace
```

Comment:
Can I make a recommendation of placing this in a separate script e.g. `ush/load_ufswm_modules.sh` and sourcing that here and in the other wave, fcst and efcs scripts instead of repeating this block everywhere?

Reply:
This is a great suggestion however some of the jobs require additional module loads that others don't and I don't know if you want the extra modules for all the jobs or not?  I guess the script could just be for the default and then added ones to the specific modules or we could have all jobs load the extra modules....   Thoughts? 

One other question I have before implementing this, it is my understanding is that this is a quick temporary fix and not expected to stay long term.  Do we want to go through the effort if it's not long term?

Reply:
IMO, it is worth the effort even if it is temporary, because we don't know how long will it take.  It also eliminates the possibility of copy/paste error.  In addition, it makes it clear which modules are being loaded and from where.

The `export USE_CFP=YES` line the `waveinit.sh` script should be in the section of `waveinit` in `env/WCOSS2.env`.
That just leaves loading the `cfp` module for wcoss2 on that job.  That can be loaded in the job in question.  It clearly identifies which modules are needed in addition to those in the ufs-weather-model.

Reply:
Thanks @aerorahul  -  I will implement these updates today.

Reply:
Updates have currently been tested on orion and wcoss2.   WCOSS2 failed in verif and I've seen others have failures in this job recently, so it's unclear if its related to this PR (i'm guessing not).  Otherwise things are fine, but the point job is taking a  longer so we either need to decrease the frequency of the point output for testing and/or increase the wall time.     Testing on hera is about to get started, but not quite sure what to do about the vrfy job though, I guess I'm going to test develop next?  Open to other ideas if anyone has them.